### PR TITLE
W-16334397 cloud connector alt text - A

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by marinasasso — DISMISSED** [review]
A few of these worry me as they look like regular steps that should be in the main text, rather than alt text for the image. I don't know the context though ...

One example:
"Configure authentication by selecting Basic in the Connection field and then complete the fields on the General tab"

File: amazon-dynamodb/1.5/modules/ROOT/pages/amazon-dynamodb-connector-studio.adoc [hunk 1]
```diff
@@ -102,7 +102,7 @@ Enter the following information on the *General* tab of the *Global Element Prop
 The following screenshot shows an example of configuring Basic authentication:
 
 .Basic authentication fields
-image::amazon-dynamodb-basic-authentication.png[To configure authentication, select `Basic` in the *Connection* field and then complete the fields on the *General* tab.]
+image::amazon-dynamodb-basic-authentication.png["Configure authentication by selecting Basic in the Connection field and then complete the fields on the General tab"]
```
Comment (sathya0) [inline amazon-dynamodb/1.5/modules/ROOT/pages/amazon-dynamodb-connector-studio.adoc]:
```suggestion
image::amazon-dynamodb-basic-authentication.png["Basic option selected in the Connection field"]
```

File: amazon-dynamodb/1.5/modules/ROOT/pages/amazon-dynamodb-connector-studio.adoc [hunk 2]
```diff
@@ -126,7 +126,7 @@ Enter the following information on the *General* tab of the global element confi
 The following screenshot shows an example of configuring Role authentication:
 
 .Role authentication fields
-image::amazon-dynamodb-role-authentication.png[To configure Role authentication, select `Role` in the *Connection* field and then complete the fields on the *General* tab.]
+image::amazon-dynamodb-role-authentication.png["Configure Role authentication by selecting Role in the Connection field and then complete the fields on the General tab"]
```
Comment (sathya0) [inline amazon-dynamodb/1.5/modules/ROOT/pages/amazon-dynamodb-connector-studio.adoc]:
```suggestion
image::amazon-dynamodb-role-authentication.png["Role option selected in the Connection field"]
```

File: amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc [hunk 3]
```diff
@@ -217,15 +217,15 @@ The following screenshots show the Anypoint Studio app flows for this example:
 
 * This flow creates the Amazon Redshift table, Amazon S3 bucket and object:
 +
-image::create-flow.png[Create the Redshift table Amazon S3 bucket and Amazon S3 object flow]
+image::create-flow.png["Create the Redshift table Amazon S3 bucket and Amazon S3 object flow"]
```
Comment (sathya0) [inline amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc]:
```suggestion
image::create-flow.png["Flow for creating the Redshift table Amazon S3 bucket and Amazon S3 object"]
```

File: amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc [hunk 4]
```diff
@@ -217,15 +217,15 @@ The following screenshots show the Anypoint Studio app flows for this example:
...
 +
 * This flow executes the `COPY` command, which leverages the Amazon Redshift massively parallel processing (MPP) architecture to load data in parallel from a file in an Amazon S3 bucket:
 +
-image::execute-flow.png[Execute the Copy command flow]
+image::execute-flow.png["Execute the Copy command flow"]
 +
 * This flow deletes the Amazon Redshift table and Amazon S3 bucket:
 +
-image:delete-flow.png[Delete the Amazon Redshift table and Amazon S3 bucket flow]
+image:delete-flow.png["Delete the Amazon Redshift table and Amazon S3 bucket flow"]
```
Comment (sathya0) [inline amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-example.adoc]:
```suggestion
image:delete-flow.png["Flow for deleting the Amazon Redshift table and Amazon S3 bucket"]
```

File: amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc [hunk 5]
```diff
@@ -71,7 +71,7 @@ To enable support for managing large message payloads in Amazon SQS Connector, f
 * *Expression or Bean Reference*
 * *Edit Inline*
 +
-image::amazon-sqs-studio-large-payload-support.png[Configure Large Payload Support]
+image::amazon-sqs-studio-large-payload-support.png["Configure Large Payload Support"]
```
Comment (sathya0) [inline amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc]:
```suggestion
image::amazon-sqs-studio-large-payload-support.png["Large Payload Support configuration"]
```

File: azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc [hunk 6]
```diff
@@ -200,14 +200,14 @@ Add File Connector to your project and configure the *Read* operation to read fi
 . Click *Add*.
 . Click *Finish*.
 +
-image::azure-data-lake-update-path-adding-file-connector.png[Add File Connector from Exchange to your flow]
+image::azure-data-lake-update-path-adding-file-connector.png["Add File Connector from Exchange to your flow"]
```
Comment (sathya0) [inline azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc]:
```suggestion
image::azure-data-lake-update-path-adding-file-connector.png["File Connector added from Exchange to your flow"]
```

File: azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc [hunk 7]
```diff
@@ -318,11 +318,11 @@ This section shows you how to implement simple error handling on a flow that use
 
 . In any flow, from the *Mule Palette* view, select *Core > On Error Continue* and drag it to the *Error Handling* section of the flow:
 +
-image::azure-data-lake-error-handling-setting-up-1.png[Drag the On Error Continue component to the flow]
+image::azure-data-lake-error-handling-setting-up-1.png["Drag the On Error Continue component to the flow"]
```
Comment (sathya0) [inline azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc]:
```suggestion
image::azure-data-lake-error-handling-setting-up-1.png["On Error Continue component dragged to the flow"]
```

File: azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc [hunk 8]
```diff
@@ -318,11 +318,11 @@ This section shows you how to implement simple error handling on a flow that use
...
 +
 . From the *Mule Palette* view, drag *Core > Transform Message* to the flow:
 +
-image::azure-data-lake-error-handling-setting-up-2.png[Drag the Transform Message component to the flow]
+image::azure-data-lake-error-handling-setting-up-2.png["Drag the Transform Message component to the flow"]
```
Comment (sathya0) [inline azure-data-lake/1.0/modules/ROOT/pages/azure-data-lake-connector-examples.adoc]:
```suggestion
image::azure-data-lake-error-handling-setting-up-2.png["Transform Message component dragged to the flow"]
```

File: azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc [hunk 9]
```diff
@@ -178,7 +178,7 @@ The following table shows the fields for configuring *Token credentials properti
 
 The following example shows the configuration for *Sas Authentication* with *Azure blob storage sas authentication* in Studio:
 
-image::aeh-sas.png[Configuring azure blob storage SAS authentication]
+image::aeh-sas.png["Configuring azure blob storage SAS authentication"]
```
Comment (sathya0) [inline azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc]:
```suggestion
image::aeh-sas.png["Configuration of azure blob storage SAS authentication"]
```

File: azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc [hunk 10]
```diff
@@ -195,7 +195,7 @@ In the *Configuration XML* editor, the `azure-eventhubs:sas-authentication-conne
 
 The following example shows the configuration for *Active directory authentication* with *Azure blob storage sas authentication* and *Token credentials properties* in Studio:
 
-image::aeh-active-directory-client.png[Configuring active directory authentication with client credentials]
+image::aeh-active-directory-client.png["Configuring active directory authentication with client credentials"]
```
Comment (sathya0) [inline azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc]:
```suggestion
image::aeh-active-directory-client.png["Configuration of active directory authentication with client credentials"]
```

File: azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc [hunk 11]
```diff
@@ -213,7 +213,7 @@ In the *Configuration XML* editor, the `azure-eventhubs:active-directory-authent
 
 The following example shows the configuration for *Active directory authentication* with *Azure blob storage active directory authentication* and *Use azure default credentials* in Studio:
 
-image::aeh-active-directory-default-client.png[Configuring active directory authentication with default client credentials]
+image::aeh-active-directory-default-client.png["Configuring active directory authentication with default client credentials"]
```
Comment (sathya0) [inline azure-event-hubs/1.1/modules/ROOT/pages/azure-event-hubs-connector-studio-configuration.adoc]:
```suggestion
image::aeh-active-directory-default-client.png["Configuration of active directory authentication with default client credentials"]
```

File: azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-studio.adoc [hunk 12]
```diff
@@ -90,7 +90,7 @@ To configure the global element for Azure Service Bus Management Connector, foll
 The following screenshot shows an example of configuring shared access signature (SAS) authentication. This screenshot uses property placeholders for field values. For information about using property placeholders, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
 +
 .SAS authentication fields
-image::azure-sb-mgmt-authentication.png[To configure SAS authentication, complete the fields on the *General* tab.]
+image::azure-sb-mgmt-authentication.png["To configure SAS authentication, complete the fields on the General tab"]
```
Comment (sathya0) [inline azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-studio.adoc]:
```suggestion
image::azure-sb-mgmt-authentication.png["SAS authentication config fields on the General tab"]
```

File: connectors-home/modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc [hunk 13]
```diff
@@ -107,7 +107,7 @@ Studio enables you to test the connection between a connector and its data sourc
 In Studio, after you configure the required connection values in the *Global Element Properties* screen, click *Test Connection*:
 
 .Click the *Test Connection* button in Studio to test the connection
-image::intro-config-overview-studio-test.png[Test the connection in Studio]
+image::intro-config-overview-studio-test.png["Test the connection in Studio"]
```
Comment (sathya0) [inline connectors-home/modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc]:
```suggestion
image::intro-config-overview-studio-test.png["Test the connection button in Studio"]
```

**Review by marinasasso — APPROVED** [review]

**Pull request merged**